templates/redis_exporter.service.j2

```diff
@@ -5,7 +5,7 @@ Requires=network-online.target
 After=network-online.target
 
 [Service]
-PrivateTmp=yes
+PrivateTmp={{ redis_exporter_private_tmp }}
```

Comment:
It would be nice to check with Molecule that this config is applied launching this command:

```$ systemctl cat redis_exporter.service```

And checking this value is equals to the value defined in test group vars

Reply:
OK, tests updated and passing